### PR TITLE
Update application config and genesis block for devnet and alphanet

### LINKED ISSUE #749: Update application config and genesis block for devnet and alphanet
### Description

Update application config and genesis block for devnet and alphanet

### Acceptance Criteria

- Update the devnet and alphanet genesis config with:
  - Reduced distance for default rewards
- Update the devnet and alphanet genesis assets with:
  - Self stakes for all the init-validators
  - Pending unlocks for stakers
  - Set commission to 50% (5000) for all the validators
- Regenerate the devnet and alphanet genesis blocks with the above changes
- Generate genesis block for betanet (without the above changes, as per existing configuration)

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #749 

### How was it solved?

- [x] Update the devnet and alphanet genesis config with:
  - [x] Reduced distance for default rewards
- [x] Update the devnet and alphanet genesis assets with:
  - [x] Self stakes for all the init-validators
  - [x] Pending unlocks for stakers
  - [x] Set commission to 50% (5000) for all the validators
- [x] Regenerate the devnet and alphanet genesis blocks with the above changes
- [x] Generate genesis block for betanet (without the above changes, as per existing configuration)
- [x] Update configurations for all networks

### How was it tested?

- ./bin/run start -n devnet  --api-ws --api-ipc -l info
-  npm run test:stress

### CODE REVIEW

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: config/devnet/genesis_assets.json [hunk 1]
```diff
@@ -2609,113 +3124,1865 @@
...
-						"lsk5fpa8xxacgv2axovx9ywxur2traz9t2ycndqz3",
-						"lsktvdgtkpv6ob86jj4fqpshuhc8gxcqnpgvghcp4",
+				"stakers": [
+					{
+						"address": "lskguo9kqnea2zsfo3a6qppozsxsg92nuuma3p7ad",
+						"stakes": [
+							{
+								"amount": "100000000",
+								"validatorAddress": "lskguo9kqnea2zsfo3a6qppozsxsg92nuuma3p7ad",
+								"sharingCoefficients": []
```
Comment (sameersubudhi) [inline config/devnet/genesis_assets.json:3139]: Same here.

File: config/devnet/genesis_assets.json [hunk 2]
```diff
@@ -2609,113 +3124,1865 @@
...
-						"lsktg3myhs8wrb7d9sckyxh8e9p8u6rt8hw3emgxs",
-						"lskffuwgvmvwje2a7zjvwxm3ctjbrjqm4q9rja2nq",
-						"lsk5fpa8xxacgv2axovx9ywxur2traz9t2ycndqz3",
-						"lsktvdgtkpv6ob86jj4fqpshuhc8gxcqnpgvghcp4",
+				"stakers": [
+					{
+						"address": "lskguo9kqnea2zsfo3a6qppozsxsg92nuuma3p7ad",
+						"stakes": [
+							{
+								"amount": "100000000",
```
Comment (sameersubudhi) [inline config/devnet/genesis_assets.json:3137]: `amount` must be at least `1000 LSK (viz. 100000000000)` and in multiples of 10 LSK (must end with atleast 9 zeros and greater than 100000000000).

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Also, please ensure that the total supply and total locked in the token module is also updated.
For calculation of the `sharingCoefficients` please refer to https://github.com/LiskHQ/lips/blob/main/proposals/lip-0070.md#efficient-calculation-of-rewards.

**Review by ManuGowda — APPROVED** [review]

**Comment by sameersubudhi** [comment]
When running Lisk Core with `devnet` config, I notice that `consecutiveMissedBlocks` is always non-zero (either 16 or 17) and the `sharingCoefficients[0].coefficient` is empty for all the validators. 

`sharingCoefficients[0].coefficient` is empty for alphanet as well. (To run alphanet, I'd to regenerate the genesis block with `modules.pos.useInvalidBLSKey: false` to bypass the current bug in SDK). However, `consecutiveMissedBlocks` was zero as expected.

Could you please confirm with the SDK/Research team if we need to fix anything at our end?

Eg (devnet):
```
    {
      "name": "genesis_43",
      "totalStake": "100000000000",
      "selfStake": "100000000000",
      "lastGeneratedHeight": 0,
      "isBanned": false,
      "reportMisbehaviorHeights": [],
      "consecutiveMissedBlocks": 16,
      "commission": 5000,
      "lastCommissionIncreaseHeight": 0,
      "sharingCoefficients": [
        {
          "tokenID": "0400000000000000",
          "coefficient": ""
        }
      ],
      "address": "lskg2ktfe4n4yd963pusqf8vbgqv7arwx9c5o8kpu",
      "punishmentPeriods": []
    }
```

**Review by sameersubudhi — APPROVED** [review]
Thank you for all the changes @nagdahimanshu. LGTM 👍

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**